extensions/wikia/AdEngine/js/ml/model/decisionTreeClassifier.js

```diff
@@ -0,0 +1,70 @@
+/*global define*/
+define('ext.wikia.adEngine.ml.model.decisionTreeClassifier', [
+	'wikia.log'
+], function () {
+	'use strict';
+
+	function buildUrl(modelId) {
+		return '/wikia.php?controller=AdEngine2Api&method=getModelData&id=' + modelId;
+	}
+
+	function findMax(nums) {
```

Comment:
What do you think about `nums.indexOf(Math.max(nums))`?

Reply:
Discussed offline